content/articles/beginners-intro-to-generative-modeling/index.md

```diff
@@ -0,0 +1,102 @@
+
+Let's try to break the above paragraph by taking a real-life example. Consider a scenario where you get appointed as a machine learning engineer in some company. Now, you give this excellent news to your parents, and now they ask you what an ML engineer is and what will be your everyday job. Let us take your information about an ML engineer as $x$. To make someone understand who has never studied ML in their life, you have to give some basic information only because too much of the technicalities will be of no use. Now you explain your job to parents in leman terms, let the information they learnt as $z$(hidden representation of actual information $x$). Now using this $z$ as an inspiration, they will draw a picture in their mind of what an ML engineer does, $x^*$. Now ideally, you want $x^*$ to be almost the same as $x$, but it will solely depend on the hidden representation of the $x$. Again, if you have to give the same information to one of your colleagues who knows ML, the $z$ will be different, and the $x^*$ will be closer to the $x$. The same applies to generative modeling. Different models will learn different hidden representations $z$ of original data, and the generated data becomes unique. 
+
+### Discriminative vs Generative Modeling
+
+Machine learning models can broadly be categorized into two categories, Discriminative and Generative Models. Before we compare both of them, let's talk about the discriminative models. In simple terms, as its name suggests, the discriminative model aims to discriminate between multiple data instances. It takes input data for training and makes predictions for the unseen data. Most of the classification and regression techniques fall under this category.
+To understand it mathematically, let's take an example of a set of data instances $X$ and a corresponding set of labels $Y$.
+
+![discriminative-vs-generative](discriminative-vs-generative.png)
+
+[Image source](https://www.analyticsvidhya.com/blog/2021/07/deep-understanding-of-discriminative-and-generative-models-in-machine-learning/)
+
+The discriminative model aims to capture $p(Y|X)$, that is, the conditional probability of $Y$ given $X$. In contrast, the generative model aims to capture joint probability $p(X, Y)$, or just $p(X)$ if there is no $Y$ present. 
+
+### Why Generative Modeling?
+
+>*The Future of AI is Generative, not Discriminative.*
+> 
+> Steve Omohundro, Research Scientist at Facebook 
+
+Generative modeling is a most underrepresented topic in machine learning. However, still, a lot of research is going on this topic. This section will discuss why generating data is essential and how it can transform the future of machine learning by taking a few interesting use-cases.
```

Comment:
How are you saying it is underrepresented? When you make such claims, you should back it up with data. If not, it is not ideal to make such claims.